base/repl/REPLCompletions.jl

```diff
@@ -239,18 +238,19 @@ function find_start_brace(s::AbstractString; c_start='(', c_end=')')
                 in_back_ticks = true
             end
         else
-            if !in_back_ticks && !in_double_quotes && c == '\'' && !done(r, i) && next(r, i)[1]!='\\'
+            if !in_back_ticks && !in_double_quotes && c == '\'' && i > 0 && s[prevind(s, i)] != '\\'
```

Comment:
It would make sense to save the result of `prevind(s, i)` since it's also used below.